nslsii/iocs/tests/test_epicsmotor_ioc.py

```diff
@@ -0,0 +1,102 @@
+import os
+import subprocess
+import sys
+import time
+
+from ophyd.epics_motor import EpicsMotor
+from ophyd.status import MoveStatus
+
+
+def test_epicsmotor_ioc():
+
+    stdout = subprocess.PIPE
+    stdin = None
+
+    '''
+    ioc_process = subprocess.Popen([sys.executable, '-m',
+                                    'caproto.tests.example_runner',
+                                    'nslsii.iocs.epics_motor_ioc_sim'],
+                                   stdout=stdout, stdin=stdin,
+                                   env=os.environ)
+    '''
```

Comment:
see comment above, why is this change necessary and can we remove the commented out code above

Reply:
See above comments associated with temperature_contollers_test